client/src/index.jsx

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import $ from 'jquery';
```

Comment:
Unless you are planning on using DOM functionality from jquery it may be better to use a smaller http-request focused package like axios or fetch

Reply:
I'll look into that next, thanks